app/uk/gov/hmrc/apidefinition/repository/APIDefinitionRepository.scala

```diff
@@ -46,6 +46,8 @@ class APIDefinitionRepository @Inject() (mongoComponent: MongoComponent)(implici
         .map(fieldName => createUniqueBackgroundSingleFieldAscendingIndex(fieldName, s"${fieldName}Index"))
     ) with Logging {
 
+  override lazy val requiresTtlIndex = false
```

Comment:
I take it they have removed this requirement.

Reply:
Have you seen the override redefinition of the stored api definition repository.  I was puzzled by this (not your change)